vital/types/track.cxx

```diff
@@ -223,6 +223,25 @@ ::find( frame_id_t frame ) const
   return this->end();
 }
 
+// ----------------------------------------------------------------------------
+bool
+track
+::contains( frame_id_t frame ) const
+{
+  if ( ( frame < this->first_frame() ) ||
```

Comment:
Why duplicate all this code from `find()` instead of just `return this->find(frame) != this->end();`?